duckduckgo/Android
# improve https upgrade performance

Asana Issue URL:  https://app.asana.com/0/488551667048375/514330959360383

## Description

Changes the https upgrade to use equality and repoints the list to a new location (https2) with a new format. 

## Steps to Test this PR:

1. When the new endpoint is available at duckduckgo.com/contentblocking.js?l=https2 launch the app (probably best if you delete any existing app first to get the new http list)
1. Allow download to occur
1. Browse to english.sogou.com - observe upgrade to https
